src/browser/squirrel-update.coffee

```diff
@@ -231,19 +129,19 @@ exports.handleStartupEvent = (app, squirrelCommand) ->
   switch squirrelCommand
     when '--squirrel-install'
       createShortcuts ->
-        installContextMenu ->
+        WinRegistry.installContextMenu ->
           addCommandsToPath ->
             app.quit()
       true
     when '--squirrel-updated'
       updateShortcuts ->
-        installContextMenu ->
+        WinRegistry.installContextMenu ->
```

Comment:
@mnquintana @paulcbetts some people mentioned that after an Atom update, the context menu would be there again, even if they manually removed by editing the registry.

Looking at code, even in this PR it seems that nothing will change on that side: the context menu will be installed again no matter what's configured in settings. Should this behaviour change as well, and take into account current settings?